### ISSUE TITLE
Plugin loaded after login fails to identify armour

### ISSUE THREAD

**Issue by Slord6** [G07cha/MineflayerArmorManager#7]
If the plugin is loaded after login, then `versionData` is not populated. I think this can be fixed by initalising `versionData` straight away, and then updating with the `login` event like so:

```javascript
var versionData = minecraftData(bot.version)

  // Version is only detected after bot logs in 
  bot.on('login', function onLogin() {
      versionData = minecraftData(bot.version)
  })
```

**Comment by G07cha** [comment]
Good point @Slord6, would you mind creating a PR for this?

**Comment by Slord6** [comment]
Yep, sure - will do so tomorrow

**Comment by G07cha** [comment]
Fix published in v1.1.0, thanks for your contribution!
